### PR TITLE
[DDI-471]Update plugin doc for version 1.1.0

### PR DESCRIPTION
**Pull request by markzegarelli**
- Added requested doc updates
- incremented plugin version global to 1.1.0

### CODE REVIEW

File: docs/session-replay/sdks/plugin.md [hunk 1]
```diff
@@ -99,6 +99,16 @@ Session replay requires that you configure default session event tracking. This
...
+!!! info "Session Start and Session End events"
+    Beginning with plugin version 1.1.0, Session Replay no longer requires `Session Start` and `Session End` events. To disable capture of these events, set `forceSessionTracking: false`:
+
+    ```js
+    const sessionReplayTracking = window.sessionReplay.plugin({ 
+        forceSessionTracking: false, 
+        sampleRate: 1, // 100% sample rate, should reduce for production traffic.  
+    }); 
+    ```
+
```
Comment (markzegarelli) [inline docs/session-replay/sdks/plugin.md:111]:
```suggestion
```

File: docs/session-replay/sdks/plugin.md [hunk 2]
```diff
@@ -187,6 +196,18 @@ Session replays may not appear in Amplitude due to:
 - No events triggered through the browser SDK in the current session
 - Sampling
 
+#### Local development and focus state
+
+The Session Replay SDK and plugin capture only the page that's in focus. When you develop locally with the browser console open, focus states may not work as expected. If you don't see replays in Amplitude, try to enable `debugMode`. In this mode, Session Replay ignores the focus handle and enables extra debugging information.
+
+
```
Comment (markzegarelli) [inline docs/session-replay/sdks/plugin.md:203]:
```suggestion
```

**Comment by katie-amplitude** [comment]
@jackson-amplitude 2 questions for you:

1 - do we need to update anything in this section? 
image

@jackson-amplitude @markzegarelli 
2 - do we need to be more explicit that these events are still captured by default? 
image

**Comment by markzegarelli** [comment]
> @jackson-amplitude 2 questions for you:
> 
> 1 - do we need to update anything in this section? image
> 
> @jackson-amplitude @markzegarelli 2 - do we need to be more explicit that these events are still captured by default? image

I will add clearer language  that speaks to that.

File: docs/session-replay/sdks/plugin.md [hunk 3]
```diff
@@ -99,6 +99,15 @@ Session replay requires that you configure default session event tracking. This
     });
     ```
 
+!!! info "Session Start and Session End events"
+    Beginning with plugin version 1.1.0, Session Replay no longer requires `Session Start` and `Session End` events. To disable capture of these events, set `forceSessionTracking: false`:
```
Comment (markzegarelli) [inline docs/session-replay/sdks/plugin.md:103]:
```suggestion
    Beginning with plugin version 1.1.0, Session Replay no longer requires `Session Start` and `Session End` events, but does capture them by default. To disable capture of these events, set `forceSessionTracking: false`:
```

**Comment by katie-amplitude** [comment]
great Mark - Jackson just let us know on my first question

**Comment by jackson-amplitude** [comment]
Rereading the doc in the first question, it's pretty confusing. If they are using a different SDK we should just point them to the standalone (the plugin is not a great use case for them). 

If you instrument your events with a different SDK (other than Browser SDK) for event instrumentation, please refer to the standalone SDK to and ensure the events are tagged with the proper event properties.

**Review by jackson-amplitude — APPROVED** [review]
Other than the comment above. Everything else looks good!

**Comment by katie-amplitude** [comment]
@jackson-amplitude are you suggesting replacing this sentence with your sentence?

current: _If you instrument your events with a method other than the Browser SDK, the Browser SDK may send only the default Session Start and Session End events._

your version: _If you instrument your events with a different SDK (other than Browser SDK) for event instrumentation, please refer to the standalone SDK to and ensure the events are tagged with the proper event properties._

**Comment by jackson-amplitude** [comment]
Yea, otherwise it's not really that useful for them with just start session events.

**Comment by katie-amplitude** [comment]
@markzegarelli you OK to make that last change?

**Review by katie-amplitude — APPROVED** [review]
great

File: docs/session-replay/sdks/plugin.md [hunk 4]
```diff
@@ -201,7 +221,7 @@ Browser extensions or network security policy may block the Session Replay SDK.
 
 #### No events triggered through the browser SDK in the current session
 
-Session Replay requires that at least one event in the user's session has the `[Amplitude] Session Replay ID` property. The [Browser SDK](/data/sdks/browser-2/) `Session Start` and `Session End` events include this property by default. If you instrument your events with a method other than the Browser SDK, the Browser SDK may send only the default `Session Start` and `Session End` events.
+Session Replay requires that at least one event in the user's session has the `[Amplitude] Session Replay ID` property. The [Browser SDK](/data/sdks/browser-2/) `Session Start` and `Session End` events include this property by default. If you instrument your events with any SDK other than the Amplitude Browser SDK 2, use the [Session Replay Standalone SDK](/session-replay/standalone) and ensure you tag your events with the necessary event properties.
```
Comment (markzegarelli) [inline docs/session-replay/sdks/plugin.md:224]:
```suggestion
Session Replay requires that at least one event in the user's session has the `[Amplitude] Session Replay ID` property. The [Browser SDK](/data/sdks/browser-2/) `Session Start` and `Session End` events include this property by default. If you instrument your events with any SDK other than the Amplitude Browser SDK 2, use the [Session Replay Standalone SDK](/session-replay/sdks/standalone) and ensure you tag your events with the necessary event properties.
```

**Pull request merged**